roadmap/implementors-guide/guide.md

```diff
@@ -1030,6 +1031,7 @@ The following conditions need to be met for an incoming statement to be accepted
 
 * The sender of the statement has to be in the validator set for the relay block.
 * If the statement is `Valid` or `Invalid`, then the candidate that the hash refers to must be known.
+* As only `MAX_CHAIN_HEADS` (currently set to 5) competing parachain heads are accepted at a time, if the statement is `Seconded`, then there must be less than `MAX_CHAIN_HEADS` other candidate receipts for the parachain block.
```

Comment:
> then there must be less than `MAX_CHAIN_HEADS` other candidate receipts for the parachain block.

This is not accurate - max chain heads is about how many relay chain heads a particular peer is allowed to say they are interested in, not about how many candidates there are for any specific relay chain head.